### PR TITLE
feat(containers): helm output

### PR DESCRIPTION
**Pull request by eladb**
Introduce an experimental parameter `provider` in `wing.toml` to allow specifying the backend for containers.

Also - fix how readiness checks are implemented.

### CODE REVIEW

File: containers/wing.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[containers]
+provider="eks"
```
Comment (MarkMcCulloh) [inline containers/wing.toml:2]: Isn't this basically what a platform is? Having this as a parameter is kinda confusing in our current model

File: containers/tfaws-eks.w [hunk 2]
```diff
@@ -23,8 +22,6 @@ interface ICluster extends std.IResource {
 pub class ClusterBase impl ICluster {
   pub attributes(): ClusterAttributes { 
     throw "Not implemented"; 
-    // WORKAROUND: Compiler doesn't recognize that this will never return, so we need to return something
-    return { certificate: "", endpoint: "", name: "" };
```
Comment (Chriscbr) [inline containers/tfaws-eks.w:27]: 😄

File: containers/tfaws-eks.w [hunk 3]
```diff
@@ -81,37 +78,36 @@ class ClusterRef extends ClusterBase impl ICluster {
...
     return existing ?? newCluster();
   }
 
-
-  static tryGetClusterAttributes(): ClusterAttributes? {
-    if !values.Values.has("eks.cluster_name") {
-      return nil;
+  static tryReadParameters(scope: std.IResource): ClusterAttributes? {
+    if let json = nodeof(scope).app.parameters.value("eks") {
+      log(json);
```
Comment (Chriscbr) [inline containers/tfaws-eks.w:104]: debugging?

File: containers/helm.w [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+    let deployment = new plus.Deployment(
+      replicas: props.replicas,
+      metadata: {
+        name: props.name
+      },
+    );
+
+    deployment.addContainer(
+      image: "\{\{ .Values.image }}",
```
Comment (Chriscbr) [inline containers/helm.w:40]:
```suggestion
      image: r"{{ .Values.image }}",
```

**Review by Chriscbr — APPROVED** [review]

File: containers/wing.toml (hunk 1 above)
Comment (eladb) [reply containers/wing.toml:2]: I agree but as long as we don't have a way to inject dependencies for non-SDK classes (https://github.com/winglang/wing/issues/4930) we need something like this in order to be able to configure the behavior of `containers.Workload`.

File: containers/tfaws-eks.w (hunk 3 above)
Comment (eladb) [reply containers/tfaws-eks.w:104]: yes

**Pull request merged**